NOTICE

```diff
@@ -23,6 +23,7 @@
 Robert Bosch GmbH
     Daniel Kunz <daniel.kunz@de.bosch.com>
     Manoranjith <ponraj.manoranjitha@in.bosch.com>
+    Dinesh Kumaran <dineshkumaran.nagarajan@in.bosch.com>
```

Comment:
@manoranjith, can you keep the list sorted alphabetically, please?

Reply:
Sorted the list in [this commit](https://github.com/direct-state-transfer/dst-go/pull/24/commits/3edaea4114f6eaa5c0deb7c51ef7eae9b3bc3e9e).